### ISSUE TITLE
I'm blocked on the step "Add the Managed Package Framework code"

### ISSUE THREAD

**Issue by dylanwa** [MicrosoftDocs/visualstudio-docs#1823]
For this step, I need to
"Set ProjectBasePath to the location of the ProjectBase.files file in the Managed Package Framework code you just downloaded"
But I didn't find "ProjectBase.files" in the download MPF code. It looks complicated to understand what it's going to do here.

Btw, this document is 2 years old. It does not work well with VS 2017. Do you have a plan to update it?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8135def2-30e0-d9e4-af54-5a8d3a073c63
* Version Independent ID: fea902ab-e883-51bd-c291-04d1988f37f6
* Content: [Creating a Basic Project System, Part 1 - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/extensibility/creating-a-basic-project-system-part-1?view=vs-2017)
* Content Source: [docs/extensibility/creating-a-basic-project-system-part-1.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/extensibility/creating-a-basic-project-system-part-1.md)
* Product: **visual-studio-dev15**
* GitHub Login: @gregvanl
* Microsoft Alias: **gregvanl**

**Comment by dulazhere** [comment]
Any update here?

**Comment by Silthrim** [comment]
The Problem seems to be the outdated MPF which is just designed for VS10-13. I found an unofficial version for VS15 and it works.
https://github.com/mimura1133/uo_mpfproj14

**Comment by leslierichardson95** [comment]
Thanks for submitting your feedback and we apologize for the delayed response.  Due to current resourcing and substantial extensibility changes we are currently working on that will result in significant, structural doc changes, we are closing this lower priority issue.  We will return to this issue on a later date and address it based on it's relevance after those extensive extensibility updates are made. Feel free to comment here as necessary.

**Comment by msweeney1999** [comment]
This is NOT a low priority issue.   Quality has really dropped since you came on board Leslie.

**Comment by WilliamAntonRohm** [comment]
@msweeney1999 -- thank you for your follow-up.  Please consider these resources:
- [Visual Studio on Microsoft Q&A](https://docs.microsoft.com/answers/products/vs) 
- [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio) 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

cc @leslierichardson95 for awareness

**Comment by HoangCaoPhi** [comment]
This is a bad guide.
